INSTRUCTION.md

```diff
@@ -0,0 +1,11 @@
+how to run MySQL container with a volume attached
+docker run -d -p 8080:8080 --name mysql-local -v mysql-volume mysql-local:1.0.0
```

Comment:
The Docker run command for the MySQL container is missing the volume specification. You should specify the volume mapping, e.g., `-v mysql-volume:/var/lib/mysql`, to ensure data persistence.